ccdproc/combiner.py

```diff
@@ -317,3 +318,206 @@ def average_combine(self, scale_func=None, scale_to=1.0):
 
         #return the combined image
         return combined_image
+
+
+def combine(img_list, output_file=None, method='average', weights=None, scale=None, mem_limit=16e9, 
+            minmax_clip=False, minmax_clip_min=None, minmax_clip_max=None, 
+            sigma_clip=False, sigma_clip_low_thresh=3, sigma_clip_high_thresh=3, 
+            sigma_clip_func=ma.mean, sigma_clip_dev_func=ma.std, **ccdkwargs):
+
+    """Convenience function for combining multiple images
+
+    Parameters
+    -----------
+    img_list : `list`, 'string'
+        A list of fits filenames or CCDData objects that will be combined together.
+        Or a string of fits filenames seperated by comma ",".
+
```

Comment:
Thank you for all the corrections. For this correction, I am not sure I understand what you meant. The code right now accepts a list of CCDData objects also, that is why I wrote "A list of fits filenames or CCDData objects"